server/endpoints.go

```diff
@@ -58,6 +58,8 @@ const (
 	EnableRuleForClusterEndpoint = "clusters/{cluster}/rules/{rule_id}/enable"
 	// MetricsEndpoint returns prometheus metrics
 	MetricsEndpoint = "metrics"
+	// RuleContent returns static content for {rule_id}
+	RuleContent = "content/{rule_id}"
```

Comment:
https://platform-docs.cloud.paas.psi.redhat.com/rest/urls.html#url-path-structure

maybe `rules/rule_id` or `rules/rule_id/content` would be slightly better with guidelines. Just a nitpick :)

Reply:
Thank you for pointing this, I wasn't aware about that guidelines, so I will change it